src/services/TrackerService.ts

```diff
@@ -107,3 +108,14 @@ export async function fileTracker(requestBody: TrackersFilePost) {
     .then(({ data }) => data)
     .catch(createCatchHandler('Failed to file tracker'));
 }
+
+export function trackerUrl(type: string, id: string): string {
+  switch (type) {
+  case 'JIRA':
+    return osimRuntime.value.backends.jira + '/browse/' + id;
+  case 'BUGZILLA':
+    return osimRuntime.value.backends.bugzilla + '/' + id;
```

Comment:
Yes, it would, but I think that as the base URLs come from `osimRuntime` we control that it doesn't have trailing /

Reply:
We do, but from a developer point of view relying on URL being with or without trailing `/` is not a very good practise.

Reply:
That's fair, I think we could make a check on `osimRuntime` and form the url with a condition based on the result.
How does it sounds to you?

Reply:
Yeah, we could probably sanitize any URL there to not contain any trailing `/`.

Reply:
One solution could be the `URL` class:
```suggestion
    return (new URL(`/browse/${id}`, osimRuntime.value.backends.jira)).href;
  case 'BUGZILLA':
    return (new URL(`/${id}`, osimRuntime.value.backends.bugzilla)).href;
```
This should handle the trailing slash, because the first parameter is an absolute path, it will replace any path the `osimRuntime.value.backends` has and only keep the hostname

Reply:
That one looks pretty clean to me @MrMarble. 
Wdyt @JakubFrejlach?

Reply:
Looking good!